packages/walletmodal/vite.config.ts

```diff
@@ -0,0 +1,44 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+import path from 'path'
+import svgr from 'vite-plugin-svgr'
+import { NodeGlobalsPolyfillPlugin } from '@esbuild-plugins/node-globals-polyfill'
+
+// https://vitejs.dev/config/
+export default () => {
```

Comment:
why do we need vite config in wallet modal package ? if this is for cypress then lets keep everything cypress related in `cypress` folder.